src/e2eFlutterTest/java/io/appium/java_client/android/CommandTest.java

```diff
@@ -115,4 +119,26 @@ void testDragAndDropCommand() {
         assertEquals(driver.findElement(AppiumBy.flutterText("The box is dropped")).getText(), "The box is dropped");
 
     }
+
+    @Test
+    void testCameraMocking() throws IOException {
+        driver.findElement(BaseFlutterTest.LOGIN_BUTTON).click();
+        openScreen("Image Picker");
+
+        final String successQr = driver.injectMockImage(
+                new File(String.valueOf(TestUtils.resourcePathToAbsolutePath("success_qr.png"))));
+        driver.injectMockImage(new File(String.valueOf(TestUtils.resourcePathToAbsolutePath("second_qr.png"))));
+
+        driver.findElement(AppiumBy.flutterKey("capture_image")).click();
+        driver.findElement(AppiumBy.flutterText("PICK")).click();
+        assertEquals(driver.findElement(AppiumBy.flutterText("SecondInjectedImage")).getText(), "SecondInjectedImage");
```

Comment:
isn't it always true? if element is found by the text, it will contain exactly the searched text, if element is not found - exception will be thrown